dashboard/app/models/user.rb

```diff
@@ -191,6 +191,8 @@ class User < ActiveRecord::Base
   accepts_nested_attributes_for :school_info, reject_if: :preprocess_school_info
   validates_presence_of :school_info, unless: :school_info_optional?
 
+  has_one :circuit_playground_discount_application
```

Comment:
I _think_ this is stricter than the `belongs_to` statement in CPDA, and maybe also makes the other redundant?  I'm not sure.

Reply:
They are inverses, the CPDA `belongs_to :user` is an association from that model to one user (or nil) based on the user_id, and this is the other way around, an association from user to CPDA, also based on CPDA's user_id. AFAIK neither is stricter than the other.

Reply:
@aoby is it correct to have the association in both directions like this, or redundant?

Reply:
Not redundant, since they are inverses. In other words, if you want to associate both directions (easily), then you need both associations.

Reply:
It would be unnecessary, but not harmful, if we were only using one.